instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/javaagent/src/test/java/io/opentelemetry/testing/instrumentation/log4j/contextdata/v2_17/AutoLog4j2Test.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.testing.instrumentation.log4j.contextdata.v2_17;
```

Comment:
usually we use the same package for code and tests, so I'd expect this class to be in `io.opentelemetry.javaagent.instrumentation.log4j.contextdata.v2_17`